README.md

```diff
@@ -19,19 +19,32 @@ environment.  After downloading the appropriate package be sure to scroll down
 to the Linux, Mac OS X, and FreeBSD tarballs section and set up your Go
 environment as per the instructions.
 
+### GCC
```

Comment:
Why GCC is required? 
Do a user need 'make'? Make is not part of GCC, in Ubuntu it is in the package 'build-essential'